examples/demo/basic/scatter_select.py

```diff
@@ -87,7 +87,6 @@ def _create_plot_component():
 # Attributes to use for the plot view.
 size = (650, 650)
 title = "Scatter plot with selection"
-bg_color = "lightgray"
```

Comment:
same comment as above - why not pass this to the `ComponentEditor`, like we do with the other examples